src/java/picard/vcf/GenotypeConcordance.java

```diff
@@ -62,14 +62,69 @@
  * @author George Grant
  */
 @CommandLineProgramProperties(
-        usage = "Calculates the concordance between genotype data for two samples in two different VCFs - one being considered the truth (or reference) " +
-                "the other being considered the call.  The concordance is broken into separate results sections for SNPs and indels.  Summary and detailed statistics are reported\n\n" +
-                "Note that for any pair of variants to compare, only the alleles for the samples under interrogation are considered " +
-                "and MNP, Symbolic, and Mixed classes of variants are not included.",
-        usageShort = "Calculates the concordance between genotype data for two samples in two different VCFs",
+        usage = GenotypeConcordance.USAGE_SUMMARY + GenotypeConcordance.USAGE_DETAILS,
+        usageShort =  GenotypeConcordance.USAGE_SUMMARY,
         programGroup = VcfOrBcf.class
 )
 public class GenotypeConcordance extends CommandLineProgram {
+    static final String USAGE_SUMMARY = "Calculates the concordance between genotype data for samples in two different" +
+            "VCFs, your callset and a standard or 'truthset'.";
+    static final String USAGE_DETAILS = "Genotype concordance is the fraction of correct assigned genotypes divided by the" +
```

Comment:
> Genotype concordance is the fraction of correct assigned genotypes divided by the" +
> +            " total number of true-positive (TP) variant alleles.  Genotype concordance is calculated by dividing the number of validated " +
> +            "genotypes in a call set by the total number of TP alleles. 

These two sentences are redundant -- the first one should be modified to explain more clearly what the GC measure aims to express.